stage0/run.go

```diff
@@ -415,6 +415,12 @@ func addResolvConf(cfg RunConfig, rootfs string) {
 	}
 	content += "\n"
 
+	if err := os.Mkdir(filepath.Join(rootfs, "etc"), defaultRegularDirPerm); err != nil {
```

Comment:
to be clear are you intentionally avoiding `MkdirAll` here? (i.e. should rootfs not existing be an error condition? arguably yes).